quickstarts/notification-channels/slack-notifications/config.yml

```diff
@@ -0,0 +1,35 @@
+id: db34c333-7b4a-4bb8-8fbf-becce43bfa44
+
+title: Slack Notifications
+
+name: slack-notifications
+description:  |
+  
+  Slack is a a messaging app for business that connects people to the information that they need.
+
+  Check out our documentation to set up a Slack notification channel and provide fast and consistent ways for the right personnel to be notified about incidents
+
+summary: |
+  Send your New Relic alerts to Slack
+
+logo: logo.png
+
+level: New Relic
+
+authors:
+  - Matan Moser (New Relic)
+  - Ismail Azam (New Relic)
+
+keywords:
+  - slack
+  - notifications
+  - AIOps
+  - Incident Intelligence
+  - Destinations
+
+documentation:
```

Comment:
Won't this add a tile to the data source tab? Which I thought we agreed would confuse the user and the idea was to just link to these doc URLS from within the markdown description.